File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.py
def solution(arr, queries):
    # for q in queries:
    #     # t=arr[q[0]]
    #     # arr[q[0]]=arr[q[1]]
    #     # arr[q[1]]=t
    #     arr[q[0]],arr[q[1]]=arr[q[1]],arr[q[0]]
    for i,j in queries:
        arr[i],arr[j]=arr[j],arr[i]
    answer=arr
    return answer

File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    int t;
    for(int i=0;i<queries.size();i++) {
        t=arr[queries[i][0]];
        arr[queries[i][0]]=arr[queries[i][1]];
        arr[queries[i][1]]=t;
    }
    return arr;
}